content/docs/api-reference/c-chain/api.mdx

```diff
@@ -81,6 +81,14 @@ For batched requests on the [public API node](/tooling/rpc-providers) , the maxi
 number of items is 40. We are working on to support a larger batch size.
 </Callout>
 
+#### Exceptions
+
+- `eth_getProof` behaves differently than geth, from release v0.14.1, with the following differences:
```

Comment:
the release version should the avalanchego version, not the coreth version. I suppose we should wait for the next avalanchego release using coreth v0.14.1 (unreleased) to merge this PR.